documentation/workflows.md

```diff
@@ -0,0 +1,74 @@
+# Working on the Project
+
+Better the be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
+
+## Principles
+
+1. No work is ever done directly on `main`; ALWAYS branch off of the target receiving branch (i.e. `main` for the majority of work)
+2. All changes must be reviewed and approved, and pass all quality gates
+
+## Development Workflows
+
+- [Starting Work](#starting-work)
+- [Pipelines](#pipelines)
+- [Changesets](#changesets)
+- [Approvals](#approvals)
+- [Local Testing](#local-testing)
+
+### Starting Work
+
+1. Pick a task from the issues/project
+    - Ask the team if anyone is working on the task
+    - Assign yourself to the taks
+2. Create a branch off of the target receiving branch (i.e. `main`)
+    - `feat/#-___` - for feature work
+    - `fix/#-___` - for hotfix work
+    - Relace "#" with the task number
+    - Replace "___" with a hyphenated breif title of the task
+3. Commit changes and push the branch
+    - If changes are being made to `src/*` files or "config" for artifact output a [Changesets](#changesets) is required.
+        - `pnpm changeset`
+        - Follow the prompts to create the changeset file for your changes.
+    - Follow the conventions of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for final (hopefully single) commit of the branch.
```

Comment:
> (hopefully single)

Reads like you are saying just one commit?

Reply:
I like to keep my PRs to a single commit for merging; easier for keeping up-to-date with `main`, easier for rollback if/when necessary. If this isn't desirable for everyone else thats fine I can remove the suggestion.

Reply:
why not just squash merge?

Reply:
I've been in situations where I wanted to move a branch - change its target/where it branches from - and it is much easier when there are fewer commits. Squash only happens when merging into a target branch.

Reply:
I wouldn't say a hard requirement. As the mantra goes "commit early, commit often". Makes it easier for you to revert things while you work and keep your branch up to date.